commons/c2cgeoportal_commons/models/main.py

```diff
@@ -1163,24 +1163,33 @@ def __unicode__(self) -> str:  # pragma: no cover
 
 class Metadata(Base):
     __tablename__ = 'metadata'
-    __table_args__ = {'schema': _schema}
+    __table_args__ = (
+        UniqueConstraint('item_id', 'name'),
```

Comment:
Constraint do no seems to be taken into account ????

Reply:
Yes, we should add a alembic script